lib/papelillo/mailer_list.ex

```diff
@@ -59,46 +59,111 @@ defmodule Papelillo.MailerList do
     end
   end
 
+  @doc """
+  Create a mailing list with the parameters.
+
+  Address must be a string without blank spaces and special characters, the domain of the email address is taken from the config.exs definition.
+  """
   def create(name, description, address, config) do
     provider = Keyword.fetch!(config, :provider)
 
-    address = address <> "@" <> Keyword.fetch!(config, :domain)
+    address =
+      Keyword.fetch(config, :domain)
+      |> case do
+        {:ok, domain} -> address <> "@" <> domain
```

Comment:
I think this looks better
```suggestion
        {:ok, domain} -> "#{address}@#{domain}"
```